mobile/native/src/channel_trade_constraints.rs

```diff
@@ -31,6 +30,48 @@ pub struct TradeConstraints {
     pub order_matching_fee_rate: f32,
     /// Total collateral in the dlc channel, none if [`is_channel_balance`] is false.
     pub total_collateral: Option<u64>,
+    /// Defines what leverage the coordinator will take depending on what the trader takes
+    ///
+    /// Unfortunately our version of flutter_rust_bridge does not support hashmaps yet
+    pub coordinator_leverages: Vec<CoordinatorLeverage>,
+    /// The leverage the coordinator will take if none is provided in `coordinator_leverages`
```

Comment:
❓Do we really need to cater for this situation by adding a default? If `coordinator_leverages` is empty I think something went very wrong and we shouldn't allow trading anyway.